<launch>
    <node pkg="arx_r5_controller" type="R5Controller" name="arm_master_l">
        <param name="arm_control_type" value="aloha_master"/>
        <param name="arm_can_id" value="can0"/>
        <param name="arm_pub_topic_name" value="/arm_master_l_status"/>
        <param name="arm_sub_topic_name" value="/arm_master_l_cmd"/>
        <param name="arm_end_type" value="1"/>
    </node>

    <node pkg="arx_r5_controller" type="R5Controller" name="arm_master_r">
        <param name="arm_control_type" value="aloha_master"/>
        <param name="arm_can_id" value="can2"/>
        <param name="arm_pub_topic_name" value="/arm_master_r_status"/>
        <param name="arm_sub_topic_name" value="/arm_master_r_cmd"/>
        <param name="arm_end_type" value="1"/>
    </node>

    <node pkg="arx_r5_controller" type="R5Controller" name="arm_slave_l">
        <param name="arm_control_type" value="aloha_slave"/>
        <param name="arm_can_id" value="can1"/>
        <param name="arm_pub_topic_name" value="/arm_slave_l_status"/>
        <param name="arm_sub_topic_name" value="/arm_master_l_status"/>
        <param name="arm_end_type" value="0"/>
    </node>

    <node pkg="arx_r5_controller" type="R5Controller" name="arm_slave_r">
        <param name="arm_control_type" value="aloha_slave"/>
        <param name="arm_can_id" value="can3"/>
        <param name="arm_pub_topic_name" value="/arm_slave_r_status"/>
        <param name="arm_sub_topic_name" value="/arm_master_r_status"/>
        <param name="arm_end_type" value="0"/>
    </node>
</launch>